index.js

```diff
@@ -1,5 +1,13 @@
 import "@tbd54566975/web5-react-native-polyfills";
 import { registerRootComponent } from "expo";
+import { polyfillBlob } from "./blob-polyfill";
+
+if (!global.structuredClone) {
+  var structuredClone = require("realistic-structured-clone");
```

Comment:
not a huge deal but we don't really want to use `var` in modern JS development

Reply:
you can probably avoid the var entirely with

`global.structuredClone = require("realistic-structured-clone")`